evergreen.yml

```diff
@@ -349,6 +364,21 @@ functions:
       - -v
       - cmake-test
 
+  ##
+  # Runs tests via ctest, ignoring flaky tests (see EVG-19909).
+  # Requires f_compile to have been run first.
+  # Feel free to update regex_clude per the latest test results
+  ##
+  f_cmake_test_ignore_flaky:
+  - command: subprocess.exec
+    params:
+      binary: ./src/genny/run-genny
+      args:
+      - -v
+      - cmake-test
+      - --regex-exclude
+      - "(sleepNonBlocking|Non-Blocking end|Actual Actor Example Timing)"
```

Comment:
Consider a standard test name e.g. IgnoredOnMacOs so the tests can be marked ignored more explicitly and we don't accidentally break CI by renaming tests.

Reply:
Just to confirm I understand you correctly, you mean renaming "Acutual Actor Example Timing" to "Actual Actor Example IgnoredOnMacOS"?

Reply:
Yes. That puts the test together with the specification to skip on macos. Again, just a suggestion.

Reply:
Good idea. It will also make the regex simpler.
Since I've already merged this PR, I'm creating a PR for this renaming.